.github/ci_templates/e2e_tests.yml

```
@@ -74,6 +75,11 @@
   LAYOUT_VSPHERE_PASSWORD: ${{ secrets.LAYOUT_VSPHERE_PASSWORD }}
   LAYOUT_VSPHERE_BASE_DOMAIN: ${{ secrets.LAYOUT_VSPHERE_BASE_DOMAIN }}
 {!{- end }!}
+  COMMENT_ID: ${{ inputs.comment_id }}
+  GITHUB_API_SERVER: ${{ github.api_url }}
+  REPOSITORY: ${{ github.repository }}
+  DHCTL_LOG_FILE: ${{ steps.setup.outputs.dhctl-log-file}}
+  TOKEN_GITHUB_BOT: ${{secrets.BOATSWAIN_GITHUB_TOKEN}}
```

I'd rather name it simply GITHUB_TOKEN.
```suggestion
  GITHUB_TOKEN: ${{secrets.BOATSWAIN_GITHUB_TOKEN}}
```